index.bs

```diff
@@ -3967,6 +3987,33 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
         interest of minimizing disruption to the user interaction flow).
 </div>
 
+### Client Capability Set (dictionary <dfn dictionary>ClientCapability</dfn>) ### {#dictionary-client-capability}
+
+<xmp class="idl">
+    dictionary ClientCapability {
```

Comment:
I wonder if there's not a reason to pluralize this, to `ClientCapabilities` instead. It's not an Enum anymore so I'd say the plural form is alright.